schema/reportdb/uyuni-reportdb-schema.changes.releng.bump

```diff
@@ -0,0 +1 @@
+- Bump version to 5.1.0
```

Comment:
For the schemas, you may want to precreate the paths from 5.0 to 5.1 for several MUs, so we always have migration paths.

Last time for 4.3 I did it from `susemanager-schema-4.3.22-to-susemanager-schema-4.4.0` (4.3.22 being the version for GA IIRC) to `susemanager-schema-4.3.59-to-susemanager-schema-4.4.0`.

Why this? Because that way you have migration paths from the 5.0 and you don't need to create it each time you create a MU for 5.0. You will only need to verify it.

This is needed for both reportdb and the regular schema.

Reply:
@mcalmer what's your take here? Considering that last time you  told  me to not create unneeded empty migrations path with  the `.gitkeep`, for the `reportdb` I only did those two that should  be needed in  the current status of 5.0 and 
 with the next tagging:
* `uyuni-reportdb-schema-5.0.6-to-uyuni-reportdb-schema-5.1.0`
* `uyuni-reportdb-schema-5.0.7-to-uyuni-reportdb-schema-5.1.0`

Reply:
This does not apply for the "major version switch". Here every future step is "needed" as you cannot know when the "older product" release the next update and switch the version one up.

Reply:
ok, more migrations paths added.